### PR TITLE
Reinvestigate Issue #828 #8604

### LINKED ISSUE #8604: Reinvestigate Issue #828
https://github.com/pimcore/pimcore/issues/828

currently breaks in combination with classification store

### PR DESCRIPTION
**Pull request by weisswurstkanone**
resolves #8604

### CODE REVIEW

File: phpstan-baseline.neon [hunk 1]
```diff
@@ -339,7 +339,10 @@ parameters:
 			message: "#^Unreachable statement \\- code above always terminates\\.$#"
 			count: 1
 			path: lib/Security/Encoder/AbstractUserAwarePasswordEncoder.php
-
+		-
+            message: "#^Call to an undefined method Symfony\\\\Component\\\\Security\\\\Core\\\\User\\\\UserInterface\\:\\:getUsername\\(\\)\\.$#"
```
Comment (dvesh3) [inline phpstan-baseline.neon]: please revert these changes and merge 10.0. thanks!

Comment (weisswurstkanone) [reply phpstan-baseline.neon]: done

**Review by dvesh3 — CHANGES_REQUESTED** [review]
password field inside localized container is still throwing exception on save 
image

```
URL: /admin/object/save?task=publish
Method: PUT
Message: Call to undefined method setLocalepass in class Pimcore\Model\DataObject\Localizedfield
Trace: 
in /home/divesh/www/dev/pimcore/pimcore/lib/Model/AbstractModel.php:257
#0 /home/divesh/www/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data/Password.php(210): Pimcore\Model\AbstractModel->__call()
#1 /home/divesh/www/dev/pimcore/pimcore/models/DataObject/Localizedfield/Dao.php(223): Pimcore\Model\DataObject\ClassDefinition\Data\Password->getDataForResource()
#2 [internal function]: Pimcore\Model\DataObject\Localizedfield\Dao->save()
#3 /home/divesh/www/dev/pimcore/pimcore/lib/Model/AbstractModel.php(246): call_user_func_array()
#4 /home/divesh/www/dev/pimcore/pimcore/models/DataObject/ClassDefinition/Data/Localizedfields.php(516): Pimcore\Model\AbstractModel->__call()
#5 /home/divesh/www/dev/pimcore/pimcore/models/DataObject/Concrete/Dao.php(257): Pimcore\Model\DataObject\ClassDefinition\Data\Localizedfields->save()
#6 /home/divesh/www/dev/pimcore/pimcore/models/DataObject/Concrete.php(223): Pimcore\Model\DataObject\Concrete\Dao->update()
#7 /home/divesh/www/dev/pimcore/pimcore/models/DataObject/AbstractObject.php(708): Pimcore\Model\DataObject\Concrete->update()
#8 /home/divesh/www/dev/pimcore/pimcore/models/DataObject/Concrete.php(777): Pimcore\Model\DataObject\AbstractObject->save()
#9 /home/divesh/www/dev/pimcore/pimcore/bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php(1283): Pimcore\Model\DataObject\Concrete->save()
#10 /home/divesh/www/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php(157): Pimcore\Bundle\AdminBundle\Controller\Admin\DataObject\DataObjectController->saveAction()
...
```
earlier reported here https://github.com/pimcore/pimcore/issues/8604#issuecomment-810333386

**Review by dvesh3 — APPROVED** [review]
